CONTRIBUTING.md

```diff
@@ -54,12 +54,14 @@ After which you can add these changes and commit again. Note that failing pre-co
 
 ## Release process
 
-Releases are simply tags on GitHub. Once changes have been merged to master:
+Releases to PyPi are handled via TravisCI and GitHub tags. Once changes have been merged to master:
 
 1. Update the version in tilesets/__init__.py
 2. Update the changelog
-3. Tag on github with `git tag`. For example `git tag -a v0.2.0 -m 'v0.2.0'`
-4. [Push new release to pypi](https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives)
+3. Commit changes to GitHub. For example `git commit -am '0.2.0' && git push origin master`
```

Comment:
I'd prefer for version bumps & changelog updates to either be in the original PR, or a follow-on PR, as opposed to committing & pushing straight to master

Reply:
Sounds fine to me. Do we want to tag from master? Ideally the tag gitsha points to a master sha.

Reply:
Updated! https://github.com/mapbox/tilesets-cli/pull/95/commits/4890fbdd841b9a2f071aa6b0b93c21a22761fc37